dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -264,9 +265,22 @@
     @JsonIgnore
     private List<Category> categories = new ArrayList<>();
 
-    @Column()
-    @Schema(description = "Short description of the entry")
-    private String topic;
+    @Column(length = 150)
```

Comment:
Code comment on reasoning for 150. Eye-balling it, it seems to match the longest topics in the zeplin prototype, but not sure if that is why